payjoin/src/receive/mod.rs

```diff
@@ -345,26 +346,46 @@ impl UncheckedProposal {
     }
 
     /// The Sender's Original PSBT
-    pub fn extract_tx_to_schedule_broadcast(&self) -> bitcoin::Transaction {
-        self.psbt.clone().extract_tx()
+    pub fn psbt_transaction(&self) -> bitcoin::Transaction { self.psbt.clone().extract_tx() }
+
+    /// The Sender's Original PSBT fee rate.
+    pub fn psbt_fee_rate(&self) -> Result<FeeRate, Error> {
```

Comment:
Not sure this function should be public, especially since it returns BadRequest, which doesn't make much sense as a return type for a fee_rate here